Apps/Admin/Server/appsettings.Development.json

```diff
@@ -15,7 +15,8 @@
     "ContentSecurityPolicy": {
         "ConnectSource": "'self' file: data: blob: filesystem: https: http: ws:",
         "FrameSource": "'self' https://dev.loginproxy.gov.bc.ca/",
-        "FrameAncestors": "'self' https://dev.loginproxy.gov.bc.ca/"
+        "FrameAncestors": "'self' https://dev.loginproxy.gov.bc.ca/",
+        "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha256-nYaPGPw9zQuQHVl09VRaW597Ut8F9Lf/VDNxGSQu3Sc=' 'sha384-/fIp3d9vqOYL5eP47pOEUYZ/h5dXL4fD9Rc4DWSuIqMZkUj+DA94qNELNYZm8VJS' 'sha384-G+v/7MQzsBBqsKVV3XKv8ThIDYIyXG6LJWqTPBrADbzQf/Ok8cTgjl0X+smak2c5' 'sha256-Tui7QoFlnLXkJCSl1/JvEZdIXTmBttnWNxzJpXomQjg=' 'sha256-vY0Cwh5hLv8sUQ61m3KQgfrqeAmcY7t6wJ2FAdYPNyU='"
```

Comment:
Should this be put into appsettings.json instead or is it specific to local dev?